docs/articles/nunit/release-notes/framework.md

```diff
@@ -5,6 +5,44 @@ uid: frameworkreleasenotes
 
 # Framework Release
 
+## NUnit 4.1 - February 23. 2024
+
+This release is a bugfix and smaller enhancement release.
+
+There are 8 issues fixed in this release.
+
+### Enhancements
+
+* [4600](https://github.com/nunit/nunit/issues/4600) Add `DateTime`/`TimeSpan` support for inequality tolerance. Thanks to [Michael Render](https://github.com/RenderMichael) for [PR 4618](https://github.com/nunit/nunit/pull/4618)
+* [4572](https://github.com/nunit/nunit/issues/4572) Make new PropertiesComparer optional. Fixed by team [PR 4608](https://github.com/nunit/nunit/pull/4608)  Adding modifier UsingPropertiesComparer() to AnyOf-, Equal-, SomeItems-, CollectionEquivalent-, CollectionSubset-, CollectionSuperSet-, DictionaryContainsKey-, DictionaryContainsKeyValuePair-, DictionaryContainsValue- and UniqueItemsConstraint.
+* [1215](https://github.com/nunit/nunit/issues/1215) Explicit specification of generic method types on TestCase and TestCaseSource. Fixed by team [PR 4620](https://github.com/nunit/nunit/pull/4620)  See [TestOf](https://docs.nunit.org/articles/nunit/writing-tests/attributes/testof.html) documentation.
```

Comment:
@OsirisTerje 
I know it is a little late, but I think the link to `TestOf`. I think we should link to one of the pages changed in https://github.com/nunit/docs/commit/20119de3ef66b1393477ebeed5d86dd1c5a62144 e.g. https://docs.nunit.org/articles/nunit/writing-tests/attributes/testcasesource.html#sources-for-generic-methods-using-testcasedata

Reply:
Thanks for noticing @mikkelbu 
I can put up a PR for that later tonight.

Reply:
@mikkelbu You're of course right, but has that been finished @stevenaw ?  Awesome if you change the RN too :-)

Reply:
They are, as part of https://github.com/nunit/docs/pull/903 :)
I can update the release notes link tonight. I can also include stable NUnit dependency in the snippets solution instead of a pre release one (unless I missed a PR where that happened already)